src/util/Helpers.cpp

```diff
@@ -123,6 +124,13 @@ bool startsWithOrContains(const QString &str1, const QString &str2,
     return str1.contains(str2, caseSensitivity);
 }
 
+bool isNeutral(const QString &s)
+{
+    const QRegularExpression re("\\p{L}");
```

Comment:
Can make this static

```suggestion
    static const QRegularExpression re("\\p{L}");
```